data_pipelines_cli/cli_constants.py

```diff
@@ -1,10 +1,18 @@
 import pathlib
 
+from data_pipelines_cli.data_structures import DataPipelinesConfig
+
 DATAHUB_URL_ENV: str = "DATAHUB_URL"
 IMAGE_TAG_TO_REPLACE: str = "<IMAGE_TAG>"
 DOCKER_REPOSITORY_URL_TO_REPLACE: str = "<DOCKER_REPOSITORY_URL>"
 INGEST_ENDPOINT_TO_REPLACE: str = "<INGEST_ENDPOINT>"
 
+DEFAULT_GLOBAL_CONFIG: DataPipelinesConfig = {
+    "username": "",
+    "templates": {},
+    "source_type": "bigquery",
+}
```

Comment:
let's skip source_type, if there won't be in init then it should be in template

how is it going to work when there is no templates? Can we provide full path to template with dp create?